### DISCUSSION TITLE
Guidance needed to integrate the PayPal

### DISCUSSION THREAD

**Discussion by nothingmuch1** [phplime/Qrexorder#14] [General]
Can you please guide me how to integrate PayPal into the subscription and checkout. Please let me know the steps how to do it.

**Comment by phplime** [comment]
if you need any help or get any bug then please mail at our support not here.
This platform is for discussing about new features and the suggestions.

**Reply by phplime** [reply]
just put your business PayPal mail in the payment settings and enable the payment that's it.
